src/components/AddressSearch/index.js

```diff
@@ -378,6 +379,14 @@ function AddressSearch({
         };
     }, []);
 
+    const filteredPredefinedPlaces = useMemo(() => {
+        if (!network.isOffline || !searchValue) {
+            return predefinedPlaces.length < 5 ? predefinedPlaces : predefinedPlaces.slice(0, 5);
+        }
+        const filterRecentPlaces = filter(predefinedPlaces, (predefinedPlace) => isSearchedLocation(searchValue, predefinedPlace));
+        return filterRecentPlaces.length < 5 ? filterRecentPlaces : filterRecentPlaces.slice(0, 5);
```

Comment:
If we're keeping 20 now, can we just show 20 to begin with and still filter it by searching? It's kind of odd to show 5, and then when you search a new result that wasn't there could pop up since it's now part of the 5 selected.

Reply:
Displaying 20 initially may cause the modal to overflow. So I reduced the amount of visible items.
![image](https://github.com/Expensify/App/assets/13826499/f0a736e8-00dc-4702-b9c0-2202ecbb462e)
Is it okay if it overflows?

Reply:
Initially, five items are displayed, but users can search anywhere else among the 20 items by typing the letters of their choice.

Reply:
Can you adjust the styling so that it doesn't overflow and becomes scrollable instead? I think that would be ideal.